### DISCUSSION TITLE
Maxmind GeoLite2-Country database not loading

### DISCUSSION THREAD

**Discussion by vectorselector** [plausible/analytics#1473] [Self-Hosted Support] [answered]
Steps: 
1) signup with Maxmind and get license key
2) edit geoip/geoip.conf and substitute values
GEOIPUPDATE_ACCOUNT_ID=123456
GEOIPUPDATE_LICENSE_KEY=mYs3cr3tK3y_whatever

$ docker-compose -f docker-compose.yml -f geoip/docker-compose.geoip.yml up  --detach --force-recreate

and yet I still see the "IP Geolocation by DB-IP" in the bottom of Countries Map
is Maxmind not loading?

**Comment by ukutaht** [comment, marked as answer]
Ah sorry that message is not very smart, it just shows for every self-hosted installation

**Reply by vectorselector** [reply]
Ahaa! Thank you!

**Comment by aozaki-kuro** [comment]
It's really confusing so I hope it could be fixed in the next release. I wasted 1 hour on this before I saw this thread...😂

**Reply by ukutaht** [reply]
Sorry about that! It should be possible to read from the database metadata which DB it is and show the relevant license notice. I've added it to my list for the next self-hosted release.

**Reply by rafalkukla** [reply]
Has been fixed?

**Reply by ruslandoga** [reply]
@rafalkukla 👋

It has been [fixed](https://github.com/plausible/analytics/issues/1804#issuecomment-1278386886) in [`v1.5.0-rc.0`](https://github.com/plausible/analytics/discussions/2324)
